packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.js

```diff
@@ -28,6 +28,9 @@ async function addWalkthrough(context, defaultValuesFilename, serviceMetadata, o
       try {
         const { add } = require('amplify-category-auth');
         await add(context);
+        context.print.warning(
+          'If a user part of a user pool group, you will need to run amplify update storage to enable IAM group policies for CRUD operations',
```

Comment:
Should it be "If a user IS part"? I would add quotes around the command for clarity.

`'If a user is part of a user pool group, you will need to run "amplify update storage" to enable IAM group policies for CRUD operations'`